pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -854,6 +903,14 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 		return nil, err
 	}
 
+	// Resize and notify the VM about changed disks
+	for _, disk := range domain.Spec.Devices.Disks {
+		if shouldExpand(disk) {
+			logger.V(1).Infof("Disk %v about to be resized by libvirt to size %d", disk, disk.Size)
+			dom.BlockResize(getSourceFile(disk), uint64(disk.Size), libvirt.DOMAIN_BLOCK_RESIZE_BYTES)
```

Comment:
do we get any return value here to know if this worked or not?

also, is this assuming the disk is a  block disk and not FS?

Reply:
Added a print, but not much I can do otherwise.
The function doesn't assume it is a block device.